packages/editor/src/client/locales/en.json

```diff
@@ -21,7 +21,7 @@
         "addTag": "Press enter to add tag",
         "allowPeerPublishing": "Allow peers to publish this article.",
         "article": "Article",
-        "articlePending": "There is already a pending publication scheduled at {{pendingPublishDate, MM/DD/YYYY HH:mm}}. Publishing again will override that publication. ",
```

Comment:
i18next already has a date and time formatting mechanism built in. You see [here](https://www.i18next.com/translation-function/formatting) which formatting is allowed and then one you want is [moment.js](https://momentjs.com/docs/#/displaying/format/). This allows you to pass date object to the `t(...)` and you can define in the translation files how that date should be displayed.

Reply:
I couldn't make the formatting work directly in the translation file so I used the same method that had been used just above on line 14 and 16 (and some other places). It also allows the use of **at** and **on** between date and time. 
Are you saying the date should get formatted for all occurences only in this translation file ?

Reply:
I changed the formatting for the built-in option.